examples/jsm/controls/OrbitControls.js

```diff
@@ -211,8 +214,12 @@ var OrbitControls = function ( object, domElement ) {
 			offset.applyQuaternion( quatInverse );
 
 			position.copy( scope.target ).add( offset );
-
-			scope.object.lookAt( scope.target );
```

Comment:
Sorry but I'm afraid your change is incorrect. I suggest you study how [Object3D.lookAt()](https://github.com/mrdoob/three.js/blob/a14f405825b938da3a1a072e2f9084441a671689/src/core/Object3D.js#L266-L306) is implemented. You should then see that your new code is not necessary.

Reply:
yeah, I was checking the lookAt implementation, which is calculating with the worldMatrix indeed. I am implementing my own controls and working out after apply this change. So guessing this should be a bug.

Reply:
The issue without the change is if you orbit around the target while you are moving the character, you wold see the target is losing focus from the camera. Or it's a desired behavior if I was misunderstanding the OrbitControl?

Reply:
@devfans See the discussion in https://github.com/mrdoob/three.js/pull/16374.